tests/backend_test_base.py

```diff
@@ -250,6 +250,8 @@ def get_shape(info):
                 return None
             return [d.dim_value if d.HasField('dim_value') else -1 for d in info.type.tensor_type.shape.dim]
         for info in model_shapes.graph.value_info:
+            if info.name == "":
```

Comment:
ONNX shouldn't be adding shape info for empty strings used to denote optional outputs, but it has a bug https://github.com/onnx/onnx/issues/3439